pkg/chunk/chunk_store.go

```diff
@@ -69,6 +70,7 @@ type StoreConfig struct {
 // RegisterFlags adds the flags required to config this to the given FlagSet
 func (cfg *StoreConfig) RegisterFlags(f *flag.FlagSet) {
 	cfg.ChunkCacheConfig.RegisterFlagsWithPrefix("", "Cache config for chunks. ", f)
+	f.BoolVar(&cfg.ChunkCacheStubs, "store.chunk-cache-stubs", false, "If true, don't write the full chunk to cache, just a stub entry.")
```

Comment:
This should only be set on the ingester correct? Would prefixing it with `ingester` make sense even though it is in the store config?

Reply:
I see what you're getting at, but not sure about the suggestion.
Yes, it's only supposed to be set when storing chunks, which we only do in the ingester today.

Reply:
Yeah, I wasn't sure of my suggestion either. I am happy with it as is.